app/src/main/java/com/github/se/icebreakrr/ui/sections/AroundYou.kt

```diff
@@ -29,11 +57,45 @@ fun AroundYouScreen(navigationActions: NavigationActions) {
             tabList = LIST_TOP_LEVEL_DESTINATIONS,
             selectedItem = navigationActions.currentRoute())
       },
+      topBar = { TopBar() },
       content = { innerPadding ->
-        // Use the innerPadding to apply padding around your content
-        Text(
-            text = "Around You screen", // TODO PLaceholder
-            modifier = Modifier.padding(innerPadding) // Applying the padding to the content
-            )
+        val context = LocalContext.current
+        if (profiles.value.isNotEmpty()) {
+          LazyColumn(
+              contentPadding = PaddingValues(vertical = 16.dp),
+              verticalArrangement = Arrangement.spacedBy(16.dp),
+              modifier = Modifier.fillMaxWidth().padding(horizontal = 8.dp).padding(innerPadding)) {
+                items(profiles.value.size) { index ->
+                  ProfileCard(
+                      profile = profiles.value[index],
+                      onclick = {
+                        Toast.makeText(context, "Profile clicked", Toast.LENGTH_SHORT).show()
+                      })
+                }
+              }
+        } else {
+          Box(
+              contentAlignment = Alignment.Center,
+              modifier = Modifier.fillMaxSize().padding(innerPadding)) {
+                Text(
+                    modifier = Modifier.padding(innerPadding).testTag("emptyProfilePrompt"),
+                    text = "There is no one around. Try moving!",
+                    fontSize = 20.sp,
+                    fontWeight = FontWeight.Bold,
+                    color = Color(0xFF575757))
+              }
+        }
+      },
+      floatingActionButton = {
+        val context = LocalContext.current
+        FloatingActionButton(
+            onClick = {
+              Toast.makeText(context, "Filter button clicked", Toast.LENGTH_SHORT).show()
```

Comment:
This filter Button is very similar from the one i have in Notification, i think we could use the same FAB for the two screens

Reply:
As I've seen your version of the fab which is far prettier than mine, could you make it a component so we can later harmonize our screens?